Source/Scene/QuadtreePrimitive.js

```diff
@@ -128,12 +133,42 @@ define([
          */
         this.tileCacheSize = defaultValue(options.tileCacheSize, 100);
 
+        /**
+         * Gets or sets the number of loading descendant tiles that is considered "too many".
+         * If a tile has too many loading descendants, that tile will be loaded and rendered before any of
+         * its descendants are loaded and rendered. This means more feedback for the user that something
+         * is happening at the cost of a longer overall load time. Setting this to 0 will cause each
+         * tile level to be loaded successively, significantly increasing load time. Setting it to a large
+         * number (e.g. 100000) will minimize the number of tiles that are loaded but tend to make
+         * detail appear all at once after a long wait.
+         * @type {Number}
+         * @default 20
+         */
+        this.loadingDescendantLimit = 20;
```

Comment:
Does this number need more tweaking?  Or are you convinced that 20 is close enough?

Reply:
I just added a Terrain Tweaks sandcastle example (under develoment) if you feel like playing with it. 20 feels about right to me, but there's no right answer, it's just balancing user feedback against performance.

Reply:
Cool, I will try to make time but no promises.